src/controllers/users_controller.ts

```diff
@@ -222,26 +216,26 @@ export const deleteOne = async (req: Request, res: Response) => {
 
     return res.status(statusCode.OK).json(resJson.success(resMessage.X_DELETE_SUCCESS(USER), user));
   } catch (err) {
-    return res.status(statusCode.INTERNAL_SERVER_ERROR).json(resJson.fail(resMessage.X_DELETE_FAIL(USER)));
+    return res.status(statusCode.INTERNAL_SERVER_ERROR).json(resJson.fail(resMessage.X_DELETE_FAIL(USER), err));
   }
 };
 
-export const createTempPassword = async (req: Request, res: Response) => {
+export const updateTempPassword = async (req: Request, res: Response) => {
   const { email } = req.body;
-  if (!email) {
-    return res.status(statusCode.BAD_REQUEST).json(resJson.fail(resMessage.NULL_VALUE));
-  }
 
   try {
     const user = await usersService.readOneByEmail(email);
     if (!user) {
       return res.status(statusCode.BAD_REQUEST).json(resJson.fail(resMessage.NO_X(USER)));
     }
 
+    const isNeverIssued = !user.tempPasswordCreatedAt ? true : false;
+    const isIssueToday = user.tempPasswordCreatedAt && user.tempPasswordCreatedAt.getDate() == new Date().getDate();
```

Comment:
```suggestion
    const isIssueToday = user.tempPasswordCreatedAt?.getDate() == new Date().getDate();
```
옵셔널 체이닝으로 줄일 수 있습니다!